runtime/service/src/main/java/org/apache/polaris/service/config/JacksonConfig.java

```diff
@@ -31,20 +31,19 @@
 import jakarta.inject.Inject;
 import jakarta.inject.Singleton;
 import org.apache.iceberg.rest.RESTSerializers;
-import org.apache.polaris.service.config.Serializers;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 @Singleton
-public class QuarkusJacksonConfig implements ObjectMapperCustomizer {
+public class JacksonConfig implements ObjectMapperCustomizer {
```

Comment:
Nomenclature:
```suggestion
public class PolarisIcebergObjectMapperCustomizer implements ObjectMapperCustomizer {
```
... because this configures the global `ObjectMapper` specific to Iceberg + inherited Polaris specifics.